bundle/jsky.app.ot/build.sbt

```diff
@@ -142,4 +142,4 @@ OsgiKeys.exportPackage := Seq(
   "jsky.science.util",
   "jsky.science.util.gui")
 
-
+fork := true
```

Comment:
It's just so you can run the test GUI and exit with **⌘+Q** without killing sbt.